source/deployment/mobile-app-deployment.rst

```diff
@@ -0,0 +1,571 @@
+
+- Why have we decided to deploy the Mattermost mobile application?
+- Who will, or should, play a key/critical role in this project?
+- Who does this project impact?
+- How much time do we have for this deployment?
+- Should, or could, an initial deployment take a simpler path?
+- Can we stagger our deployment, starting with simple requirements first?
+- Are you going to  run a pilot deployment, if so who with?
+- What  IT or HR policies that need to change?
+
+We recommend setting at least one, achievable and time-based outcome.
+
+At the same time, you should begin thinking about technical and security requirements. A great place to start is with an implementation plan. So, feel free to `use our template available <https://docs.mattermost.com/getting-started/implementation_plan.html>`_ in the Mattermost Documentation. Also, begin by engaging with your technical and security questions like those below.
+
+- Are there any known security or access requirements?
+- Is a VPN connection required?
+- Do we use an Enterprise Mobile Management (EMM) Provider?
+
+Again, this means pausing before you jump into your Mattermost mobile deployment. This takes a little more time, but will make the difference in ensuring your see a return on your investment.
+
+----
+
+###################################
+Mattermost Mobile Deployment Primer
+###################################
+
+A mobile deployment is not a small project. While Mattermost provides options to make it as easy as possible, it will take time and iteration.
+
+We’re here to support you.
+
+This primer serves as an introduction to Mattermost Mobile technologies. It is a compliment to our technical documentation, and focuses on three main areas:
+
+- Technical Requirements
+- Basic Features and Technologies
+- Trouble Spots
+
+Regardless of your expertise, we recommend reading this section before starting your deployment.
+
+***********************
+Technical  Requirements
+***********************
+
+`Supported Mattermost Server Versions <https://github.com/mattermost/mattermost-mobile/blob/master/CHANGELOG.md>`_ 
+    Minimum requirements for the Mattermost Server is maintained in our documentation (link above). 
+    **However, we do recommend running the latest version** of the Mattermost Server as it contains the most recent features and any applicable security updates.
+
+    If this is not possible, we encourage you to bon at least version 5.12 of the Mattermost Server. This release has a number of critical feature updates that will ensure compatibility in a number of areas, including the Mattermost push notification service (MPNS).
+
+`Supported Devices/Mobile Device Requirements <https://docs.mattermost.com/install/requirements.html#mobile-apps>`_
+    Basic mobile device requirements are provided in the link above.
+
+`Network Requirements <https://>`_
+
+********************************
+Public App Store vs. Custom Apps
```

Comment:
Public App Store in my mind only references the Apple App Store, maybe consider something like
```suggestion
Mattermost distributed vs. Custom Builds
```

Reply:
“App Stores” is directly from the developer docs. We can strike “public”. But I’d like to keep consistent with that language. For reference — “You can build the app from source and distribute it within your team or company either using the App Stores, Enterprise App Stores or EMM providers, or another way of your choosing.” — https://developers.mattermost.com/contribute/mobile/build-your-own/

Reply:
For some additional reading "app store" is an accepted generic (ref:https://en.wikipedia.org/wiki/Google_Play). I get how this might cause confusion, technically it's right. I'll leave that to the group, but it would require doc changes to match.

Reply:
This seems like a shift from the original comment, "Public App Store in my mind only references the Apple App Store, maybe consider something like."

Reply:
nope, is not.. App Store references the Apple App Store, I'm just pointing out why the word `public` should not be used.

Reply:
solved in meeting